backend/docs/multi_pathogen.md

```diff
@@ -0,0 +1,454 @@
+      configFile:
+        #          TODO what do we need here? This is what was used in #4570:
+#        taxon_id: 12059 # All EVs A = 138948
+#        scientific_name: "Enterovirus"
+#        molecule_type: "genomic RNA"
+    referenceGenomes:
+      CV-A10:
+        nucleotideSequences:
+          - name: main
+            sequence: "..."
+            insdcAccessionFull: ...
+        genes: 
+          - name: VP4
+            sequence: "..."
+          - ...
+      EV-A71:
+        nucleotideSequences:
+          - name: main
+            sequence: "..."
+            insdcAccessionFull: ...
+        genes:
+          - name: VP2
+            sequence: "..."
+          - ...
+      # ...
+```
+
+The backend and the website will then receive the `referenceGenomes` as configured above.
+LAPIS will receive a "merged" reference genome 
+that merges all suborganisms by prepending the suborganism name to the sequence name 
+(ignoring `main` if there is only a single segment):
+```json
+{
+  "nucleotideSequences": [
+    {"name": "CV-A10", "sequence": "..."},
+    {"name": "EV-A71", "sequence": "..."},
+    // ...
+  ],
+  "genes": [
+    {"name": "CV-A10-VP4", "sequence": "..."},
+    {"name": "EV-A71-VP2", "sequence": "..."},
+    // ...
+  ]
+}
+```
+
+Consequence:
+* In the single-segmented case, the backend will also remain single-segmented for every suborganism. 
+  That case con be recognized by taking a set of all nucleotide sequence names 
+  and checking whether it contains only one element.
+* The reference genomes of the website and the backend will be different from the LAPIS reference genome.
+  (In the single pathogen case, they will still be the same.)
+
+## Submission
+
+Metadata: as usual, nothing special here. (To be confirmed, is that true?)
+
+Sequences: We don't want the user to be forced to put the segment name here.
+
+```
+>key1
+ACTG
+```
+
+For multiple segments:
+
+```
+>key1_segment1
+ACTG
+>key1_segment2
+GTCA
+```
+
+TODO: Is the `fileMapping` relevant here?
+
+### INSDC Ingest
+
+No changes here.
+
+## Storage of unprocessed sequences
+
+```json
+{
+    "submissionId": "key1",
+    "accession": "PP_0EYHTR4",
+    "version": 1,
+    "original_data": {
+        "metadata": {...},
+        "unalignedNucleotideSequences": {
+            "main": "ACTG"
+        }
+    }
+}
+```
+
+Multi-segments:
+
+```json
+{
+    "submissionId": "key1",
+    "accession": "PP_0EYHTR4",
+    "version": 1,
+    "original_data": {
+        "metadata": {...},
+        "unalignedNucleotideSequences": {
+            "segment1": "ACTG",
+            "segment2": "GTCA"
+        }
+    }
+}
+```
+
+## preprocessing input
+
+Similar to what is stored in the DB.
+
+## Preprocessing
+
+TODO: Minimizer index, etc?
+
+## Preprocessing output
+
+```json
+{
+  "accession": "key1",
+  "version": 1,
+  "data": {
+    "metadata": {...},
+    "unalignedNucleotideSequences": {
+      "matchingSuborganism": "ACTG",
+      // omit the non-matching suborganisms
```

Comment:
We currently send a list of all segments to the backend where a segment is None when it does not exist in the data

Reply:
As discussed: We'll keep the requirement that the preprocessing sends all required segments.